src/runtime_src/xdp/profile/plugin/aie_profile/util/aie_profile_util.h

```diff
@@ -44,6 +44,15 @@ namespace xdp::aie::profile {
     XAIE_MEM_MOD
   };
 
+  // Stream width in bits
+  const std::unordered_map<uint8_t, uint8_t> streamWidthMap = {
+    {XAIE_DEV_GEN_AIE,     32},
+    {XAIE_DEV_GEN_AIEML,   32}
+    // {XAIE_DEV_GEN_AIE2P,   64}
+    // {XAIE_DEV_GEN_AIE2PS,  64}
```

Comment:
Is there a reason why these are commented out?

Reply:
These definitions are yet not available in xaiegbl_defs.h , I will have to follow up with driver team to have it populated.
Currently it is used by AIE1/AIE2